openfe/setup/methods/rbfe_methods.py

```diff
@@ -241,7 +270,11 @@ def run(self) -> bool:
         return False
 
     def is_complete(self) -> bool:
-        return False
+        results_file = self._settings.SimulationLengthSettings.output_filename
+
+        # TODO: Can improve upon this by checking expected length of the
```

Comment:
Unfortunately not as simple because you might terminate early if you're doing online analysis.

_should_ be feasible to look at the settings and assess the outcome though.